### PR TITLE
OCPBUGS-39133: Configure graceful shutdown for metrics-server

### PR DESCRIPTION
**Pull request by benluddy**
* [ ] I added CHANGELOG entry for this change.
* [ ] No user facing changes, so no entry in CHANGELOG was needed.

### CODE REVIEW

**Comment by dgrisonnet** [comment]
Holding since this requires to update the fork to include https://github.com/kubernetes-sigs/metrics-server/pull/1560

**Comment by simonpasquier** [comment]
/retitle OCPBUGS-39133: Configure graceful shutdown for metrics-server

**Review by simonpasquier — COMMENTED** [review]
/lgtm
/approve

File: jsonnet/components/metrics-server.libsonnet [hunk 1]
```diff
@@ -199,6 +199,9 @@ function(params) {
                 '--tls-cert-file=/etc/tls/private/tls.crt',
                 '--tls-private-key-file=/etc/tls/private/tls.key',
                 '--tls-cipher-suites=' + cfg.tlsCipherSuites,
+                '--shutdown-send-retry-after=true',
+                // wait long enough for the readiness probe's failure threshold to be breached
+                '--shutdown-delay-duration=150',
```
Comment (simonpasquier) [inline jsonnet/components/metrics-server.libsonnet:204]:
```suggestion
                '--shutdown-delay-duration=150s',
```

**Pull request merged**